identity_core/Cargo.toml

```diff
@@ -26,7 +26,7 @@ zeroize = { version = "1.4", default-features = false }
 [dependencies.iota-crypto]
 version = "0.12.1"
 default-features = false
-features = ["ed25519", "random", "sha", "x25519"]
+features = ["ed25519", "random", "sha", "x25519", "std"]
```

Comment:
Need this in order for `crypto::Error` to implement `std::error::Error` so we can use the `#[source]` annotation with the wrapped variant.